src/Waives.Http/ConsoleLogProvider.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Waives.Http.Logging;
+using Waives.Http.Logging.LogProviders;
+
+namespace Waives.Http
+{
+    public class ConsoleLogProvider : LogProviderBase
```

Comment:
Is it worth adding colouring support to this provider, so the messages are easy to differentiate? Given this is a built-in provider, I feel as though it should provide user-friendly output.

Reply:
Also needs an XML doc comment, and should be marked `sealed` to prevent people deriving from it. It would be worth calling this out as for dev/test only, not for production use. I have no idea what the performance of `ReplaceStructuredLoggingTokens()` will be in real life.